common/src/main/java/io/seata/common/loader/EnhancedServiceLoader.java

```diff
@@ -115,11 +114,12 @@
      * @return the s
      * @throws EnhancedServiceNotFoundException the enhanced service not found exception
      */
-    public static <S> S load(Class<S> service, String activateName, Object[] args) throws EnhancedServiceNotFoundException {
+    public static <S> S load(Class<S> service, String activateName, Object[] args)
+        throws EnhancedServiceNotFoundException {
```

Comment:
Why change the line here ？
I think we should have more characters on each line to avoid more line breaks.